tools/analyzer_plugin/lib/src/diagnostic/missing_required_prop.dart

```diff
@@ -190,27 +217,27 @@ class MissingRequiredPropDiagnostic extends ComponentUsageDiagnosticContributor
         }),
       );
     }
-  }
 
-  static String _requiredPropsDebugMessage(RequiredPropInfo requiredPropInfo) {
```

Comment:
This method got inlined/indented, and is mostly the same; see [whitespace-agnostic diff](https://github.com/Workiva/over_react/pull/897/files?w=1#diff-5c58cc83ab2e904cbe44afe6887d688d66d4435a1c26684d359f6cf7f30d57e1R222).